genie-agent/src/test/groovy/com/netflix/genie/agent/execution/services/impl/grpc/GRpcAgentFileStreamServiceImplSpec.groovy

```diff
@@ -467,20 +573,20 @@ class GRpcAgentFileStreamServiceImplSpec extends Specification {
 
                 @Override
                 void onNext(final AgentManifestMessage value) {
-                    printf("Received manifest")
+                    println("Received manifest")
```

Comment:
Shouldn't these be `log` messages instead of straight `println`?

Reply:
It could, but at the moment all messages in groovy spock tests are printed with println